src/pages/pupils/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug]/index.tsx

```diff
@@ -94,7 +107,7 @@ const PupilsPage: NextPage<PupilLessonOverviewPageProps> = ({
   return (
     <OakThemeProvider theme={oakDefaultTheme}>
       <LessonEngineProvider>
-        <OakBox $height={"100vh"} $minWidth={"100vw"}>
```

Comment:
This was causing the page to scroll horizontally unnecessarily when there was vertical scrolling. The `100vw` element is inside a scrolling ancestor (the document root) causing horizontal scrolling equal to the width of the scrollbar 

<img width="304" alt="Screenshot 2024-02-07 at 09 15 02" src="https://github.com/oaknational/Oak-Web-Application/assets/122096/f2ed64ea-cb1d-4297-8d3f-7e1e343a97dc">